test/unit/util.test.ts

```diff
@@ -60,3 +60,12 @@ test('Encoding ArticleId for Zim HTML Url', async(t) => {
         t.equal(encoded, encodeArticleIdForZimHtmlUrl(unencoded), `encodeArticleIdForZimHtmlUrl() encoding`);
     }
 });
+
+async function testHtmlRewritingE2e(t: any, wikicode: string, html: string, comment: string){
+    const resultHtml = await convertWikicodeToHtml(wikicode, 'https://en.wikipedia.org/');
+    t.equal(html, resultHtml.data, comment);
+}
+
+test('wikitext comparison', async(t) => {
+    testHtmlRewritingE2e(t, `An [[isolated system]] remains the system is free.`, `<p id="mwAQ">An <a rel="mw:WikiLink" href="./Isolated_system" title="Isolated system" id="mwAg">isolated system</a> remains the system is free.</p>`, 'HTML and Wikitext match')
```

Comment:
Try to avoid lines of code with more than 80 characters. Here the readability would be improved with somethin like:
```
testHtmlRewritingE2e(
  t,
  `An [[isolated system]] remains the system is free.`,
  `<p id="mwAQ">An <a rel="mw:WikiLink" href="./Isolated_system" title="Isolated system" id="mwAg">isolated system</a> remains the system is free.</p>`,
  'HTML and Wikitext match')
```